jc/parsers/net_localgroup.py

```diff
@@ -0,0 +1,208 @@
+r"""jc - JSON Convert `net localgroup` command output parser
+
+
+Usage (cli):
+
+    $ net localgroup | jc --net-localgroup
+    $ net localgroup /domain | jc --net-localgroup
+    $ net localgroup Administrators | jc --net-localgroup
+    $ net localgroup Administrators /domain | jc --net-localgroup
+
+Usage (module):
+
+    import jc
+    result = jc.parse('net-localgroup', net_localgroup_command_output)
```

Comment:
Though 'net-localgroup' will work here, it is best to use 'net_localgroup' for python module examples.